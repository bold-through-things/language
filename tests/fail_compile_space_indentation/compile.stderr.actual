[
  {
    "message": "print statement has no children",
    "line": 2,
    "content": "print"
  }
]
