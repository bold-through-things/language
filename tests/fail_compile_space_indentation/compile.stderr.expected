[
  {
    "recoverable": false,
    "line": 3,
    "char": 0,
    "content": "    string hello world",
    "error": "this language only accepts tabs for indentation, not spaces! spaces are like, totally uncool. use tabs instead, they're way more precise and semantic.",
    "error_type": "INVALID_INDENTATION"
  }
]